.claude/skills/youtube-collector/README.md

```
@@ -0,0 +1,107 @@
+```
+
+### 2. API 키 설정
+```bash
+python3 .claude/skills/youtube-collector/scripts/setup_api_key.py
+```
+
+## 사용 프롬프트 예시
+
+### 채널 등록
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)